filebeat/docs/inputs/input-journald.asciidoc

```diff
@@ -284,7 +284,8 @@ available:
 `CONTAINER_ID_FULL`::         `container.id`
 `CONTAINER_NAME`::            `container.name`
 `CONTAINER_PARTIAL_MESSAGE`:: `container.partial`
-`CONTAINER_TAG`::             `container.image.tag`
+`CONTAINER_TAG`::             `container.log.tag`
```

Comment:
Just to be sure to understand, what is the reason to switch from `container.image.tag` to `container.log.tag`?

Reply:
It's just a matter of correcting the docs. The actual field is called [`container.log.tag`](https://github.com/elastic/beats/blob/4504968401342a289e443e94a2d8547496170bef/filebeat/input/journald/pkg/journalfield/default.go#L80)

Reply:
Make sense thanks for the explanation! 
Looks good to be merged then!